libs/langchain/langchain/schema/runnable/fallbacks.py

```diff
@@ -34,13 +34,61 @@
 
 
 class RunnableWithFallbacks(RunnableSerializable[Input, Output]):
-    """
-    A Runnable that can fallback to other Runnables if it fails.
+    """A Runnable that can fallback to other Runnables if it fails.
+
+    External APIs (e.g., APIs for a language model) may at times experience
+    degraded performance or even downtime.
+
+    In these cases, it can be useful to have a fallback runnable that can be
+    used in place of the original runnable (e.g., fallback to another LLM provider).
+
+    Fallbacks can be defined at the level of a single runnable, or at the level
+    of a chain of runnables. Fallbacks are tried in order until one succeeds or
+    all fail.
+
+    While you can instantiate a ``RunnableWithFallbacks`` directly, it is usually
+    more convenient to use the ``with_fallbacks`` method on a runnable.
+
+    Example:
+
+        .. code-block:: python
+
+            from langchain.chat_models.openai import ChatOpenAI
+            from langchain.chat_models.anthropic import ChatAnthropic
+
+            model = ChatAnthropic().with_fallbacks([ChatOpenAI()])
```

Comment:
nit but may be more interesting to fallback for prompt | model, since usually you'll want model-specific prompt and it's not obvious that you can use fallbacks for that

Reply:
Got it -- we can follow up with another example. I think the first one should be very simple (no chains involved).